server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -657,15 +786,56 @@ public void start() {
         blobStoreRepository = (BlobStoreRepository) repository;
     }
 
+    private ClusterMetadataManifest uploadV1Manifest(
```

Comment:
why would we need V1 support?

Reply:
As of now this code is not used anywhere, kept this in case we make codec version configurable. Do you think we can remove this now and add it back when needed later?

Reply:
we should be able to read V0 & V1 but for write we can switch to V2 completely unless you are thinking to provide a flag to switch to v1 for writing as well?

Reply:
We don't have any such plans as of now. So, I have removed this method.